.gitignore

```diff
@@ -88,8 +88,10 @@ examples/bin
 .vs
 TestResults/*
 .vscode/*
+**/.vscode/*
```

Comment:
I'm surprised this is different than the below rule? Do we need both?

Reply:
this handles the scenario if I create a workspace on a subfolder, say "sdk/iot/modelsrepository" (so i'm not working on the top level and have to deal with all the other libraries). The rule above does not handle this scenario.

Reply:
I meant the below rule of: `./**/.vscode/*` which seems to be "any .vscode folder underneath the root" -- I didn't think the leading `./` would change anything

Reply:
gotcha, not sure about that. I'll tinker and see why I felt the need to add it when I did. But honestly this shouldn't be in this PR even if it is helpful because it's adding in a different change. I'll remove it.